### PR TITLE
ci: check for stdlib includes (Closed #32)

### LINKED ISSUE #32: check bloat includes in CI
my clangd keeps adding ``#include <stdint.h>`` the CI should catch those outside of common.h

### PR DESCRIPTION
**Pull request by ChillerDragon**
Yoinked from https://github.com/ddnet/ddnet/blob/787f85c25de77d6fd7abf6f7327ce0089a3fd4d3/scripts/check_standard_headers.sh And adjusted for our use case

### CODE REVIEW

**Comment by MilkeeyCat** [comment]
I don't understand why the script looks for C++ header files, the ones without `.h`. C compiler doesn't understand them.

**Comment by ChillerDragon** [comment]
My goofy ah clangd adds the C++ header when I type out stuff like ``uint8_t``
But yes you are right it is covered by the compiler. So that would be covered by the build pipeline already.
So I removed the C++ headers now.

File: scripts/check_standard_headers.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# List of C and C++ headers
```
Comment (MilkeeyCat) [inline scripts/check_standard_headers.sh:3]: List of C headers

**Comment by MilkeeyCat** [comment]
clangd suggests C++ headers probably because tests are written in C++ and CMake does some magic with it

**Comment by MilkeeyCat** [comment]
Thanks

**Pull request merged**